docker-compose.yml

```diff
@@ -30,9 +30,18 @@ services:
       - "6080:6080"
 
   gargoyle-sts:
-    image: kr7ysztof/gargoyle-sts:0.0.2
+    image: kr7ysztof/gargoyle-sts:0.0.3
     environment:
       - STS_HOST=0.0.0.0
       - STS_PORT=12345
+      - GARGOYLE_KEYKLOAK_PUBLIC_KEY_ID=FJ86GcF3jTbNLOco4NvZkUCIUmfYCqoqtOQeMfbhNlE
+      - GARGOYLE_KEYKLOAK_URL=http://127.0.0.1:8080
+    network_mode: host
+
+  keycloak:
+    image: kr7ysztof/keycloak-it:0.0.1
+    environment:
+      - KEYCLOAK_USER=admin
+      - KEYCLOAK_PASSWORD=admin
     ports:
-      - 12345:12345
+      - "8080:8080"
```

Comment:
Let's move the s3 proxy to a different port than 8080 in the reference.conf. Now these conflict

Reply:
done -  8987